calico-cloud/release-notes/index.mdx

```diff
@@ -111,7 +111,7 @@ We've made it easier for platform operators to share Image Assurance scan result
 - Export one row per image or one row per image and CVE.
 - Export CSV or JSON files.
 
-To get started see, [Image Assurance](../image-assurance/scan-image-registries.mdx).
+To get started see, [Image Assurance](../image-assurance).
```

Comment:
@penkeysuresh one thing you should check with the docs team about is whether we should add the release notes in this PR or wait for a separate that will aggregate the release items.

Reply:
@penkeysuresh For CC, RNs are usually a separate PR because PM/Erik needs to collect the items for SaaS and CE/GA releases. However, if this is the only item that needs to be updated as part of this R2 release, I see no reason why you could not add it to this PR.

Reply:
> updated as part of this R2 release

@bmckercher123 you mean R1 release, right?

Reply:
@penkeysuresh it probably makes more sense to wait for a separate PR to include in-cluster scanner release notes. 

With this PR we can "stealth" launch the user docs.

Reply:
yes we are planning to have another PR for the release notes.